### PR TITLE
Adding RELEASE.md for 2.22.0 / 22.06

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

**Comment by tanmayv25** [comment]
LGTM.. But we might have to add the pytorch custom ops known issue depending upon our success with the issue.

**Comment by mc-nv** [comment]
> LGTM.. But we might have to add the pytorch custom ops known issue depending upon our success with the issue.
It's not final, I will take lates version before publish

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+* Python backend models now support 
+  [auto-generated model configuration](https://github.com/triton-inference-server/python_backend/tree/r22.06#auto_complete_config).
+
+* [Decoupled API](https://github.com/triton-inference-server/server/blob/r22.06/docs/decoupled_models.md#python-model-using-python-backend) 
+  support in Python Backend model is out of beta.
+
+* Updated I/O tensors 
+  [naming convention](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#special-conventions-for-pytorch-backend) 
+  for serving TorchScript models via PyTorch backend.
```
Comment (tanmayv25) [inline RELEASE.md:45]: Should also add =>
"Improvements to Perf Analyzer stability and profiling logic."

Comment (mc-nv) [reply RELEASE.md:45]: I have added missed statement to the documentation, thanks for the catch

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+* Triton cannot retrieve GPU metrics with MIG-enabled GPU devices (A100 and A30).
+
+* Triton metrics might not work if the host machine is running a separate DCGM 
+  agent on bare-metal or in a container.
+
+* Running a PyTorch TorchScript model using the PyTorch backend, where multiple 
+  instances of a model are configured can lead to a slowdown in model execution 
+  due to the following PyTorch issue: 
+  [pytorch/pytorch#27902](http://pytorch/pytorch#27902).
```
Comment (Jackiexiao) [inline RELEASE.md:86]: the correct links is https://github.com/pytorch/pytorch/issues/27902

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+* Improvements to Perf Analyzer stability and profiling logic.
+
+## Known Issues
+
+* Perf Analyzer stability criteria has been changed which may result in 
+  reporting instability for scenarios that were previously considered stable. 
+  This change has been made to improve the accuracy of Perf Analyzer results. 
+  If you observe this message, it can be resolved by increasing the 
+  `--measurement-intervalin` the time windows mode or
```
Comment (tanmayv25) [inline RELEASE.md:59]: measurement-intervalin -> measurement-interval

Comment (tanmayv25) [reply RELEASE.md:59]: `--measurement-interval` in the time

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+* Triton metrics might not work if the host machine is running a separate DCGM 
+  agent on bare-metal or in a container.
+
+* Running a PyTorch TorchScript model using the PyTorch backend, where multiple 
+  instances of a model are configured can lead to a slowdown in model execution 
+  due to the following PyTorch issue: 
+  [pytorch#27902](https://github.com/pytorch/pytorch/issues/27902).
+
+* Starting in 22.02, the Triton container, which uses the 22.06 PyTorch
```
Comment (CoderHam) [inline RELEASE.md:88]: * 22.02 PyTorch

File: README.md [hunk 5]
```diff
@@ -74,6 +74,10 @@ Major features include:
 - [Metrics](docs/metrics.md) indicating GPU utilization, server
   throughput, server latency, and more
 
+Need enterprise support?  NVIDIA global support is available for Triton 
+Inference Server with the 
+[NVIDIA AI Enterprise software suite](https://www.nvidia.com/en-us/data-center/products/ai-enterprise/).
```
Comment (CoderHam) [inline README.md:79]: Remove trailing "/"

Comment (CoderHam) [reply README.md:79]: NVIDIA AI Enterprise Software Suite

File: README.md [hunk 6]
```diff
@@ -103,6 +107,10 @@ regarding this example. The quickstart guide also contains an example of how to
 
 ## Examples and Tutorials
 
+Check out [NVIDIA LaunchPad](https://www.nvidia.com/en-us/data-center/products/ai-enterprise-suite/trial/)
```
Comment (CoderHam) [inline README.md:110]: Remove trailing "/"

File: README.md [hunk 7]
```diff
@@ -182,7 +190,7 @@ provide APIs to simplify this communication.
 
 ### Extend Triton
 
-[Triton Inference Server's archicture](docs/architecture.md) is specifically 
+[Triton Inference Server's architecture](docs/architecture.md) is specifically 
 designed for modularity and flexibility
```
Comment (CoderHam) [inline README.md:194]: Add trailing "."

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (mc-nv) [reply README.md:110]: This file is cherry-picked from `main` branch and it's a part of different commit, can cause a merge conflicts later.

File: RELEASE.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+* Triton metrics might not work if the host machine is running a separate DCGM 
+  agent on bare-metal or in a container.
+
+* Running a PyTorch TorchScript model using the PyTorch backend, where multiple 
+  instances of a model are configured can lead to a slowdown in model execution 
+  due to the following PyTorch issue: 
+  [pytorch#27902](https://github.com/pytorch/pytorch/issues/27902).
+
+* Starting in 22.02, the Triton container, which uses the 22.05 PyTorch
```
[the remark this answers is not in the document]
Comment (tanmayv25) [reply RELEASE.md:88]: It should be `22.06 PyTorch container`.

[the remark this answers is not in the document]
Comment (mc-nv) [reply RELEASE.md]: Updated per documentation suggestion

File: README.md (hunk 5 above)
Comment (mc-nv) [reply README.md:79]: This file is cherry-picked from `main` branch and it's a part of different commit, can cause a merge conflicts later.

**Comment by dzier** [comment]
Misha, the commit name MUST be the same name as in the release docs. Can you rename the commit to "Update README for xx.yy release" as stated in the release documentation. Be sure to name all the commits correctly for parts of the release phases.

File: RELEASE.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+  [auto-generated model configuration](https://github.com/triton-inference-server/python_backend/tree/r22.06#auto_complete_config).
+
+* [Decoupled API](https://github.com/triton-inference-server/server/blob/r22.06/docs/decoupled_models.md#python-model-using-python-backend) 
+  support in Python Backend model is out of beta.
+
+* Updated I/O tensors 
+  [naming convention](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#special-conventions-for-pytorch-backend) 
+  for serving TorchScript models via PyTorch backend.
+
+* Refer to the 22.06 column of the
```
Comment (tanmayv25) [inline RELEASE.md:47]: Just make this the last bullet point of the section. Rest looks good.

File: README.md (hunk 5 above)
Comment (dzier) [reply README.md:79]: Do not change this at all. It has been committee approved and worded. Leave it as it is please.

File: README.md (hunk 7 above)
Comment (dzier) [reply README.md:194]: No, none of the other points have that. Please keep review to only lines changed.

**Review by dzier — CHANGES_REQUESTED** [review]

**Review by tanmayv25 — APPROVED** [review]

**Comment by mc-nv** [comment]
> Misha, the commit name MUST be the same name as in the release docs. Can you rename the commit to "Update README for xx.yy release" as stated in the release documentation. Be sure to name all the commits correctly for parts of the release phases.

@dzier  it's done, commit message set to the standard.

**Review by CoderHam — APPROVED** [review]

**Review by dzier — APPROVED** [review]

**Pull request merged**